src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java

```diff
@@ -393,10 +393,12 @@ else if (isPropertyTokenType) {
         return result;
     }
 
-    private static String emptyStringArrayDefaultValue(Node defaultValueNode) {
+    private static String emptyStringArrayDefaultValue(Node defaultValueNode,
+                                                boolean isPropertyTokenType) {
         String defaultValueText = getNodeText(defaultValueNode);
         if ("{@code {}}".equals(defaultValueText)
-            || "{@code all files}".equals(defaultValueText)) {
+            || "{@code all files}".equals(defaultValueText)
+            || isPropertyTokenType && "{@code empty}".equals(defaultValueText)) {
```

Comment:
`{@code empty}` above you removed the only occurrence of this in code, why it is added here?

Reply:
this code do comparison of Xdoc (where we have "empty") with javadoc (where we have "").
So we need to keep xdco and javadoc in state as they are, so comparator code should be updated to treat them the same.